Textile/Classes/Services/TTELifecycleManager.h

```diff
@@ -0,0 +1,15 @@
+#import "TextileApi.h"
+
+#ifndef TTELifecycleManager_h
+#define TTELifecycleManager_h
+
+@protocol TTELifecycleManager <NSObject>
```

Comment:
This is the protocol that can be implemented to manage starting and stopping the node. The one implementation I made uses app state events and a timer to achieve that, but another useful implementation may be one that simply passes calls to start and stop directly through to the node. This could be used in a app share extension, for example, because it wouldn't rely on UIKit. cc @Gozala.